development/compiling/compiling_with_mono.rst

```diff
@@ -0,0 +1,66 @@
+.. _doc_compiling_with_mono:
+
+Compiling with Mono
+===================
+
+.. highlight:: shell
+
+Requirements
+------------
+
+-  Mono 5.2.0+
```

Comment:
Maybe add before the list a sentence to say that the platform-specific information on compiling Godot also apply here, and that the following instructions should be followed after checking the relevant tutorials on <code>:ref:\`Compiling \<toc-devel-compiling\>\`</code>.